packages/material-ui/src/IconButton/IconButton.js

```diff
@@ -103,7 +104,21 @@ IconButton.propTypes = {
   /**
    * The icon element.
    */
-  children: PropTypes.node,
+  children: chainPropTypes(PropTypes.node, props => {
+    React.Children.toArray(props.children).map(child => {
+      if (child.props.onClick) {
```

Comment:
Is `child.props` always defined here? It's not obvious to me that react would set a `props` property on a string child.